### ISSUE TITLE
error-is-as: `assert.CollectT`'s `Errorf()` is incorrectly reported and converted into `ErrorIsf()`

### ISSUE THREAD

**Issue by pmalek** [Antonboom/testifylint#231]
### Problem statement

`assert.CollectT`'s `Errorf()` is incorrectly reported as broken and `--fix` converts it into `ErrorIsf()` which fails to compile.

Full example:

```
package main

import (
	"strconv"
	"testing"
	"time"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
)

func TestX(t *testing.T) {
	require.EventuallyWithT(t,
		func(c *assert.CollectT) {
			_, err := strconv.Atoi("a")
			if err != nil {
				c.Errorf("failed: %v", err)
				return
			}
		},
		time.Second,
		time.Millisecond,
	)
}
```

### Proposed solution

Do not convert `assert.CollectT`'s `Errorf()` calls to `ErrorIsf()`.

**Comment by Antonboom** [comment]
hi, @pmalek 

thank you for request
I did not met `assert.CollectT` earlier, I'll know!

supported in other places too.
